Transient Scan ID: 3677     Time: 2025-10-17 23:10:08
Persistent Unique Scan ID: 'd3ce2ff4-f841-4366-97de-14c7037b7655'
I Fri-23:10:15.309: Finished SAXS/WAXS data collection in 6.65 seconds.
I Fri-23:10:15.356: Collected WAXS with HDF5 file: /share1/USAXS_data/2025-10/10_15_FanLinkam/MSI07/MSI07_waxs/MSI07_841C_187min_1311.hdf
I Fri-23:10:15.371: Collecting USAXS for MSI07_841C_187min
I Fri-23:10:15.395: Moving to USAXS mode ... please wait ...
I Fri-23:10:22.500: Ensuring technique directory exists: /share1/USAXS_data/2025-10/10_15_FanLinkam/MSI07/MSI07_usaxs
I Fri-23:10:22.500: Flyscan HDF5 data file: /share1/USAXS_data/2025-10/10_15_FanLinkam/MSI07/MSI07_usaxs MSI07_841C_187min_1312.h5
I Fri-23:10:28.350: Measured USAXS transmission values : Diode = 1313839 with gain 1e+06 and I0 = 537192 with gain 1e+08
I Fri-23:10:28.880: Ensuring technique directory exists: /share1/USAXS_data/2025-10/10_15_FanLinkam/MSI07/MSI07_usaxs
E Fri-23:10:59.861: Run aborted
Traceback (most recent call last):
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py", line 1609, in _run
    msg = self._plan_stack[-1].send(resp)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/apstools/plans/labels_to_streams.py", line 242, in label_stream_wrapper
    return (yield from bpp.plan_mutator(plan, action[when.value]))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 181, in plan_mutator
    raise ex
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 134, in plan_mutator
    msg = plan_stack[-1].send(ret)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 1375, in __call__
    return (yield from plan)
            ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 1228, in baseline_wrapper
    return (yield from plan_mutator(plan, insert_baseline))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 181, in plan_mutator
    raise ex
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 134, in plan_mutator
    msg = plan_stack[-1].send(ret)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 845, in monitor_during_wrapper
    return (yield from plan2)
            ^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 181, in plan_mutator
    raise ex
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 134, in plan_mutator
    msg = plan_stack[-1].send(ret)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 181, in plan_mutator
    raise ex
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 134, in plan_mutator
    msg = plan_stack[-1].send(ret)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 907, in fly_during_wrapper
    return (yield from plan2)
            ^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 181, in plan_mutator
    raise ex
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 134, in plan_mutator
    msg = plan_stack[-1].send(ret)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 181, in plan_mutator
    raise ex
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 134, in plan_mutator
    msg = plan_stack[-1].send(ret)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams11/USAXS/bits_usaxs/src/usaxs/user/linkam.py", line 736, in Fan625Plan
    yield from collectAllThree(isDebugMode)
  File "/home/beams11/USAXS/bits_usaxs/src/usaxs/user/linkam.py", line 689, in collectAllThree
    yield from USAXSscan(pos_X, pos_Y, thickness, sampleMod, md={})
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 1258, in dec_inner
    return (yield from plan)
            ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 448, in suspend_wrapper
    return (yield from finalize_wrapper(_inner_plan(), _remove()))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 536, in finalize_wrapper
    ret = yield from plan
          ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 446, in _inner_plan
    return (yield from plan)
            ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 1258, in dec_inner
    return (yield from plan)
            ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 448, in suspend_wrapper
    return (yield from finalize_wrapper(_inner_plan(), _remove()))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 536, in finalize_wrapper
    ret = yield from plan
          ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 446, in _inner_plan
    return (yield from plan)
            ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 1952, in __iter__
    return (yield from self._iter)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams11/USAXS/bits_usaxs/src/usaxs/plans/plans_usaxs.py", line 109, in USAXSscan
    yield from Flyscan(x, y, thickness_mm, title, md=_md)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 1952, in __iter__
    return (yield from self._iter)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams11/USAXS/bits_usaxs/src/usaxs/plans/plans_usaxs.py", line 604, in Flyscan
    yield from Flyscan_internal_plan(md=_md)  # flyscan proper
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 1952, in __iter__
    return (yield from self._iter)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams11/USAXS/bits_usaxs/src/usaxs/plans/fly_scan_plan.py", line 200, in Flyscan_internal_plan
    yield from bps.open_run(md=_md)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 1952, in __iter__
    return (yield from self._iter)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/plan_stubs.py", line 1352, in open_run
    return (yield Msg("open_run", **(md or {})))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 89, in plan_mutator
    msg = plan_stack[-1].throw(exception)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 192, in single_gen
    return (yield msg)
            ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 206, in plan_mutator
    inner_ret = yield msg
                ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 206, in plan_mutator
    inner_ret = yield msg
                ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 89, in plan_mutator
    msg = plan_stack[-1].throw(exception)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 192, in single_gen
    return (yield msg)
            ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 206, in plan_mutator
    inner_ret = yield msg
                ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 206, in plan_mutator
    inner_ret = yield msg
                ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 89, in plan_mutator
    msg = plan_stack[-1].throw(exception)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 192, in single_gen
    return (yield msg)
            ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 206, in plan_mutator
    inner_ret = yield msg
                ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 89, in plan_mutator
    msg = plan_stack[-1].throw(exception)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 192, in single_gen
    return (yield msg)
            ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py", line 206, in plan_mutator
    inner_ret = yield msg
                ^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py", line 1676, in _run
    new_response = await coro(msg)
                   ^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py", line 1879, in _open_run
    new_uid = await current_run.open_run(msg)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/bundlers.py", line 111, in open_run
    await self.emit(DocumentNames.start, doc)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py", line 2679, in emit
    self.emit_sync(name, doc)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py", line 2676, in emit_sync
    self.dispatcher.process(name, doc)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py", line 2699, in process
    exceptions = self.cb_registry.process(name, name.name, doc)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 422, in process
    func(*args, **kwargs)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py", line 512, in __call__
    return mtd(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/databroker/v1.py", line 968, in insert
    self._serializer(name, doc)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/suitcase/mongo_normalized/__init__.py", line 148, in __call__
    return super().__call__(name, sanitized_doc)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/event_model/__init__.py", line 191, in __call__
    return self._dispatch(name, doc, validate)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/event_model/__init__.py", line 199, in _dispatch
    output_doc = getattr(self, name)(doc)
                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/suitcase/mongo_normalized/__init__.py", line 254, in start
    self._insert("start", doc)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/suitcase/mongo_normalized/__init__.py", line 152, in _insert
    self._collections[name].insert_one(doc)
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/collection.py", line 891, in insert_one
    self._insert_one(
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/collection.py", line 831, in _insert_one
    self._database.client._retryable_write(
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 2040, in _retryable_write
    return self._retry_with_session(retryable, func, s, bulk, operation, operation_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 1926, in _retry_with_session
    return self._retry_internal(
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/_csot.py", line 119, in csot_wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 1972, in _retry_internal
    ).run()
      ^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 2709, in run
    return self._read() if self._is_read else self._write()
                                              ^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 2819, in _write
    self._server = self._get_server()
                   ^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 2802, in _get_server
    return self._client._select_server(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py", line 1791, in _select_server
    server = topology.select_server(
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py", line 409, in select_server
    server = self._select_server(
             ^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py", line 387, in _select_server
    servers = self.select_servers(
              ^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py", line 294, in select_servers
    server_descriptions = self._select_servers_loop(
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams/USAXS/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py", line 344, in _select_servers_loop
    raise ServerSelectionTimeoutError(
pymongo.errors.ServerSelectionTimeoutError: dbbluesky1.xray.aps.anl.gov:27017: [Errno 111] Connection refused (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68f0ebaaab3c71e69ef44832, topology_type: Single, servers: [<ServerDescription ('dbbluesky1.xray.aps.anl.gov', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('dbbluesky1.xray.aps.anl.gov:27017: [Errno 111] Connection refused (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
E Fri-23:11:30.441: Failed to close run <bluesky.bundlers.RunBundler object at 0x7f819263b990>.
Traceback (most recent call last):
  Cell In[70], line 1
    RE(Fan625Plan(0,0,0.135,"MSI07",841,200,720,40,200))
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:973 in __call__
    plan_return = self._resume_task(init_func=_build_task)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:1120 in _resume_task
    raise exc
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:1755 in _run
    raise err
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:1609 in _run
    msg = self._plan_stack[-1].send(resp)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/apstools/plans/labels_to_streams.py:242 in label_stream_wrapper
    return (yield from bpp.plan_mutator(plan, action[when.value]))
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:181 in plan_mutator
    raise ex
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:134 in plan_mutator
    msg = plan_stack[-1].send(ret)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:1375 in __call__
    return (yield from plan)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:1228 in baseline_wrapper
    return (yield from plan_mutator(plan, insert_baseline))
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:181 in plan_mutator
    raise ex
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:134 in plan_mutator
    msg = plan_stack[-1].send(ret)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:845 in monitor_during_wrapper
    return (yield from plan2)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:181 in plan_mutator
    raise ex
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:134 in plan_mutator
    msg = plan_stack[-1].send(ret)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:181 in plan_mutator
    raise ex
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:134 in plan_mutator
    msg = plan_stack[-1].send(ret)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:907 in fly_during_wrapper
    return (yield from plan2)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:181 in plan_mutator
    raise ex
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:134 in plan_mutator
    msg = plan_stack[-1].send(ret)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:181 in plan_mutator
    raise ex
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:134 in plan_mutator
    msg = plan_stack[-1].send(ret)
  File /home/beams11/USAXS/bits_usaxs/src/usaxs/user/linkam.py:736 in Fan625Plan
    yield from collectAllThree(isDebugMode)
  File /home/beams11/USAXS/bits_usaxs/src/usaxs/user/linkam.py:689 in collectAllThree
    yield from USAXSscan(pos_X, pos_Y, thickness, sampleMod, md={})
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:1258 in dec_inner
    return (yield from plan)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:448 in suspend_wrapper
    return (yield from finalize_wrapper(_inner_plan(), _remove()))
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:536 in finalize_wrapper
    ret = yield from plan
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:446 in _inner_plan
    return (yield from plan)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:1258 in dec_inner
    return (yield from plan)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:448 in suspend_wrapper
    return (yield from finalize_wrapper(_inner_plan(), _remove()))
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:536 in finalize_wrapper
    ret = yield from plan
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:446 in _inner_plan
    return (yield from plan)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:1952 in __iter__
    return (yield from self._iter)
  File /home/beams11/USAXS/bits_usaxs/src/usaxs/plans/plans_usaxs.py:109 in USAXSscan
    yield from Flyscan(x, y, thickness_mm, title, md=_md)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:1952 in __iter__
    return (yield from self._iter)
  File /home/beams11/USAXS/bits_usaxs/src/usaxs/plans/plans_usaxs.py:604 in Flyscan
    yield from Flyscan_internal_plan(md=_md)  # flyscan proper
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:1952 in __iter__
    return (yield from self._iter)
  File /home/beams11/USAXS/bits_usaxs/src/usaxs/plans/fly_scan_plan.py:200 in Flyscan_internal_plan
    yield from bps.open_run(md=_md)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:1952 in __iter__
    return (yield from self._iter)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/plan_stubs.py:1352 in open_run
    return (yield Msg("open_run", **(md or {})))
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:89 in plan_mutator
    msg = plan_stack[-1].throw(exception)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:192 in single_gen
    return (yield msg)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:206 in plan_mutator
    inner_ret = yield msg
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:206 in plan_mutator
    inner_ret = yield msg
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:89 in plan_mutator
    msg = plan_stack[-1].throw(exception)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:192 in single_gen
    return (yield msg)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:206 in plan_mutator
    inner_ret = yield msg
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:206 in plan_mutator
    inner_ret = yield msg
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:89 in plan_mutator
    msg = plan_stack[-1].throw(exception)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:192 in single_gen
    return (yield msg)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:206 in plan_mutator
    inner_ret = yield msg
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:89 in plan_mutator
    msg = plan_stack[-1].throw(exception)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:192 in single_gen
    return (yield msg)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/preprocessors.py:206 in plan_mutator
    inner_ret = yield msg
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:1676 in _run
    new_response = await coro(msg)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:1879 in _open_run
    new_uid = await current_run.open_run(msg)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/bundlers.py:111 in open_run
    await self.emit(DocumentNames.start, doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:2679 in emit
    self.emit_sync(name, doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:2676 in emit_sync
    self.dispatcher.process(name, doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/run_engine.py:2699 in process
    exceptions = self.cb_registry.process(name, name.name, doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:422 in process
    func(*args, **kwargs)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/bluesky/utils/__init__.py:512 in __call__
    return mtd(*args, **kwargs)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/databroker/v1.py:968 in insert
    self._serializer(name, doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/suitcase/mongo_normalized/__init__.py:148 in __call__
    return super().__call__(name, sanitized_doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/event_model/__init__.py:191 in __call__
    return self._dispatch(name, doc, validate)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/event_model/__init__.py:199 in _dispatch
    output_doc = getattr(self, name)(doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/suitcase/mongo_normalized/__init__.py:254 in start
    self._insert("start", doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/suitcase/mongo_normalized/__init__.py:152 in _insert
    self._collections[name].insert_one(doc)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/collection.py:891 in insert_one
    self._insert_one(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/collection.py:831 in _insert_one
    self._database.client._retryable_write(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:2040 in _retryable_write
    return self._retry_with_session(retryable, func, s, bulk, operation, operation_id)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:1926 in _retry_with_session
    return self._retry_internal(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/_csot.py:119 in csot_wrapper
    return func(self, *args, **kwargs)
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:1972 in _retry_internal
    ).run()
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:2709 in run
    return self._read() if self._is_read else self._write()
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:2819 in _write
    self._server = self._get_server()
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:2802 in _get_server
    return self._client._select_server(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/mongo_client.py:1791 in _select_server
    server = topology.select_server(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py:409 in select_server
    server = self._select_server(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py:387 in _select_server
    servers = self.select_servers(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py:294 in select_servers
    server_descriptions = self._select_servers_loop(
  File ~/.conda/envs/bits_usaxs/lib/python3.11/site-packages/pymongo/synchronous/topology.py:344 in _select_servers_loop
    raise ServerSelectionTimeoutError(
ServerSelectionTimeoutError: dbbluesky1.xray.aps.anl.gov:27017: [Errno 111] Connection refused (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 68f0ebaaab3c71e69ef44832, topology_type: Single, servers: [<ServerDescription ('dbbluesky1.xray.aps.anl.gov', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('dbbluesky1.xray.aps.anl.gov:27017: [Errno 111] Connection refused (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>


In